### PR TITLE
Update all services

### PR DESCRIPTION
**Pull request by AdyenAutomationBot**
OpenAPI spec or templates produced changes on 27-02-2024   by [commit](https://github.com/Adyen/adyen-openapi/commit/4a17315624d653e6b486a1e1aa7f8627b52f2e28).

### CODE REVIEW

**Comment by DjoykeAbyah** [comment]
⚠️  Classname change and some field name changes ⚠️ 

Changed from PINFunctionalityApi to ManageCardPINApi

**Review by DjoykeAbyah — DISMISSED** [review]

File: src/typings/balancePlatform/transferRoute.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@
  */
 
 import { AddressRequirement } from './addressRequirement';
+import { AddressRequirement | AmountMinMaxRequirement | BankAccountIdentificationTypeRequirement | PaymentInstrumentRequirement } from './addressRequirement | AmountMinMaxRequirement | BankAccountIdentificationTypeRequirement | PaymentInstrumentRequirement';
```
[the remark this answers is not in the document]
Comment (DjoykeAbyah) [reply src/typings/balancePlatform/transferRoute.ts:11]: -_-'

[the remark this answers is not in the document]
Comment (wboereboom) [reply src/typings/balancePlatform/transferRoute.ts:11]: Does this work?

[the remark this answers is not in the document]
Comment (jillingk) [reply src/typings/balancePlatform/transferRoute.ts:11]: No it clashes with line 10

**Comment by naseemkullah** [comment]
Any ETA of when this will be released? need to use `terminalInstructions.adyenAppRestart` for terminal-settings.
IF not any suggested workaround?

I've tried this but see no `terminalInstructions` in the response, so unclear if it works:

```ts
 await mgmtApi.TerminalSettingsStoreLevelApi.updateTerminalSettingsByStoreId(
      storeId,
      {
        ...terminalSettings,
        terminalInstructions: {adyenAppRestart: true},
      } as TerminalSettings
    );
 ```

**Comment by wboereboom** [comment]
Hi @naseemkullah ,
Sorry for the inconvenience. We're dealing with some hiccups in code generation as you might have been able to tell. We're working around this manually so we can provide you the necessary fixes before the end of the day.

Kind Regards,
Wouter
Adyen

**Review by DjoykeAbyah — APPROVED** [review]

**Comment by naseemkullah** [comment]
> Hi @naseemkullah ,
> 
> Sorry for the inconvenience. We're dealing with some hiccups in code generation as you might have been able to tell. We're working around this manually so we can provide you the necessary fixes before the end of the day.
> 
> 
> 
> Kind Regards,
> 
> Wouter
> 
> Adyen

Amazing - much appreciated! 🙏

Good luck with the tooling issues!

**Pull request merged**